crates/hulk_replayer/src/worker_thread.rs

```diff
@@ -1,36 +1,73 @@
 use std::{
     thread::spawn,
-    time::{Duration, SystemTime},
+    time::{Duration, Instant, SystemTime},
 };
 
 use tokio::{runtime::Builder, select, sync::watch, time::sleep};
 
 use crate::{execution::Replayer, ReplayerHardwareInterface};
 
+#[derive(Clone, Copy)]
+pub struct PlayerState {
+    pub time: SystemTime,
+    pub playing: bool,
+    pub playback_rate: f32,
+}
+
+impl Default for PlayerState {
+    fn default() -> Self {
+        Self {
+            time: SystemTime::UNIX_EPOCH,
+            playing: Default::default(),
```

Comment:
Write explicit `false` to make more visible what the default is?

Reply:
Done